bash/bashrc

```diff
@@ -43,26 +43,6 @@ source /usr/local/bin/virtualenvwrapper.sh
 export EDITOR=$(which vim);
 alias no-pyc-files="find . -name '*.pyc' | xargs rm -f"
 
-# For the lulz.
-alias sudo="echo 'With great power comes great responsibility.'; sudo"
```

Comment:
yeah this stuff should go in my aliases I guess, or maybe even a personal branch.